addons/l10n_fr_account/models/account_move.py

```diff
@@ -6,6 +6,13 @@ class AccountMove(models.Model):
 
     l10n_fr_is_company_french = fields.Boolean(compute='_compute_l10n_fr_is_company_french')
 
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'FR':
```

Comment:
See that little field l10n_fr_is_company_french, maybe we could use that ? You can tell me why it would be better ? :eyes: